shell/platform/tizen/channels/platform_view_channel.cc

```diff
@@ -174,6 +168,7 @@ void PlatformViewChannel::HandleMethodCall(
     };
     auto it = view_instances_.find(viewId);
     if (viewId >= 0 && it != view_instances_.end()) {
+      it->second->SetFocus(false);
```

Comment:
What's the difference between SetFocus(false) and ClearFocus()? It seems a little ambiguous

Reply:
In the case like ClearFocus(), it is used to reset IMF on Android App. Actually, we don't reset IMF when receiving the event at the moment. And SetFocus() is just used to check that a current platform view is focused or not.